cvtoolkit/datasets/yolo_labels_dataset.py

```diff
@@ -105,12 +119,20 @@ def get_txt_files(self):
     def get_filtered_labels(self):
         return self._filtered_labels
 
-    def _prepare_labels(self):
+    def _filter_bboxes_by_conf(self, bboxes: npt.NDArray, conf: float):
+        # Confidence scores are the sixth column, if there are less we don't
+        # have conf scores so we don't filter.
+        if bboxes.shape[1] < 6:
+            return bboxes
+        else:
+            return bboxes[bboxes[:, 5] >= conf]
+
+    def _prepare_labels(self, confidence_threshold: Optional[float] = None):
         """
         Loop through the yolo labels and store them in a dict.
 
-        Each key in the dict is an image, each value is a ndarray (n_detections, 5)
-        The 6 columns are in the yolo format, i.e. (target_class, x_c, y_c, width, height)
+        Each key in the dict is an image, each value is a ndarray (n_detections, 6)
+        The 6 columns are in the yolo format, i.e. (target_class, x_c, y_c, width, height, conf)
```

Comment:
This can be with or without confidence now, right? 
it should be explained here that it can be 5 or 6